docs/updating/nim-migration/online-builder.md

```diff
@@ -0,0 +1,169 @@
+
+The npm package is the most flexible and powerful way to install CKEditor&nbsp;5. It allows you to create a custom build of the editor with only the features you need, thus significantly reducing the final size of the build. However, you will need a JavaScript bundler or meta-framework to create such a build.
+
+If you do not want to have a build process, you can either use our CDN build or download the ZIP archive. Both of these include the editor and all plugins, so you can use all the features of CKEditor&nbsp;5 without setting up a build process.
+
+## CDN build
+
+The CDN build is a good option if you want to quickly add CKEditor&nbsp;5 to your website without having to install any dependencies or set up a build process. We recommend using our new interactive [Builder](https://ckeditor.com/ckeditor-5/builder?redirect=docs) to customize the build to your needs. Then, in the `Installation` section of the Builder you can select the `Cloud (CDN)` option to learn how to add the editor to your website.
+
+## ZIP archive
+
+If you do not want to have a build process or use our CDN build, you can download the ZIP archive from the [CKEditor&nbsp;5 download page](https://ckeditor.com/ckeditor-5/download/#zip). This archive contains the editor build with all of its plugins, which you can extract and include on your website.
+
+## npm package
+
+If you decide to use the npm package, you can either use our new interactive [Builder](https://ckeditor.com/ckeditor-5/builder?redirect=docs) to create a new build, or you can update your existing project from the legacy Online Builder. **We recommend using the new interactive Builder**, but if you want to keep your existing build, you can follow the steps below.
+
+1. Follow the steps in the {@link updating/nim-migration/customized-builds Migrating from customized builds} guide.
+	
+2. Once this is done, remove the old `build' folder and run the following command to create a new build of CKEditor&nbsp;5.
+
+		```bash
+		npm run build
+		```
+
+3. There should be three files in the new `build` folder:
+
+			* `ckeditor.d.ts`,
+			* `ckeditor.js`,
+			* `ckeditor.js.map`.
+
+		Now you can start to remove some unused webpack plugins and update the `webpack.config.js` file.
+
+4. Uninstall the following `devDependencies`:
+
+		```bash
+		npm uninstall \
+			@ckeditor/ckeditor5-dev-translations \
+			@ckeditor/ckeditor5-dev-utils \
+			@ckeditor/ckeditor5-theme-lark\
+			css-loader \
+			postcss \
+			postcss-loader \
+			raw-loader \
+			style-loader \
+			terser-webpack-plugin
+		```
+
+5. Install the following packages:
+
+		```bash
+		npm install --save-dev \
+			css-loader \
+			css-minimizer-webpack-plugin \
+			mini-css-extract-plugin \
+			terser-webpack-plugin
+		```
+
+6. Update the `webpack.config.js` file:
+
+		```js
+		/**
+		* @license Copyright (c) 2014-2024, CKSource Holding sp. z o.o. All rights reserved.
```

Comment:
The second year could use some automation, but I'm not sure this is available.

Reply:
I asked, but unfortunately there no variable we could use.

Reply:
eee, why do we place it in the docs?

Reply:
Yeah, I also started wondering if it's needed. WDYT about removing this license comment?

Reply:
yes, we can remove it. It's just a "quote" in the docs.

Reply:
Done :heavy_check_mark: